src/lib.rs

```diff
@@ -206,6 +206,17 @@ impl OptUdeps {
 		mut stdout :W,
 		clap_matches :&ArgMatches
 	) -> CargoResult<i32> {
+		if self.verbose > 0 {
+			let mut shell = config.shell();
+			shell.warn(
+				"currently verbose command informations (\"Running `..`\") are not correct.",
+			)?;
+			shell.warn("`cargo-udeps` modify the commands as follows:")?;
```

Comment:
1. it should be "modifies" not "modify"
2. the documentation is nice but I'd prefer a phrasing that makes this a non-guarantee: "for example, `cargo-udeps` does these modifications:". That way, if we change something and don't update the documentation, it's not such a big deal.

Reply:
Ok.